iscroll/v4/index.d.ts

```diff
@@ -1,76 +0,0 @@
-// Type definitions for iScroll 4.2
```

Comment:
It's valid to leave v4 in. This way it can receive bugfixes. Consider un-deleting please.